services/wordpress/wordpress-base.js

```diff
@@ -13,6 +13,7 @@ const themeSchema = Joi.object()
     num_ratings: nonNegativeInteger,
     downloaded: nonNegativeInteger,
     active_installs: nonNegativeInteger,
+    requires: stringOrFalse.required(),
```

Comment:
Just want to confirm this key will always exist for themes? Know we had an overly strict schema issue here recently but can't remember if that was due to `requires` not existing or string vs string/bool

Reply:
It should always exist. If it has an actual value, it will return it, if not it will return false.

I think you may be thinking of issue #5499, which was a bug caused by the old schema only expecting a string and breaking if the theme wasn't set to usable data. That should have been resolved with PR #5579 which introduced the stringOrFalse.